app/controllers/api/users_controller.rb

```diff
@@ -50,17 +53,7 @@ def user_notifications
   	render json: notifications
   end
 
-  def read_notification
-    updated = Notification.find(params[:notif_id]).update({read: true})
-
-    if updated
-      render json: {message: 'success'}
-    else
-      render json: {message: 'fail'}
-    end
-  end
-
-  def read_all_notifications
+  def read_notifications
```

Comment:
The "Mark as Read" functionality for Notifications is broken because you renamed `read_all_notifications` to `read_notifications`, you can either fix this by changing the name back or by updating places that use this to the new name.